booster/test/rpc-integration/resources/implies.kore

```diff
@@ -0,0 +1,2685 @@
+[topCellInitializer{}(LblinitGeneratedTopCell{}()), org'Stop'kframework'Stop'attributes'Stop'Source{}("Source(/private/tmp/pytest-of-sam/pytest-8/popen-gw1/kompiled0/k-files/definition-1.k)")]
```

Comment:
Maybe we should check in the original K file for this.
Ditto for `implies2.{k,kore}`

Reply:
These are generated via pyk, so not sure how to go about extracting a K definition